numba/unicode.py

```diff
@@ -382,6 +382,15 @@ def _find(substr, s):
     return -1
 
 
+@njit(_nrt=False)
+def _rfind(substr, s):
```

Comment:
May be it is better to put this code into `find` directly?

Reply:
I agree it make sense.